fcrepo-fixity/src/main/java/org/fcrepo/camel/fixity/FixityRouter.java

```diff
@@ -37,42 +38,48 @@
 
     private static final String REPOSITORY = "http://fedora.info/definitions/v4/repository#";
 
+    @Autowired
+    private FcrepoFixityConfig config;
+
     /**
      * Configure the message route workflow.
      */
     public void configure() throws Exception {
-
         final Namespaces ns = new Namespaces("rdf", RDF.uri);
         ns.add("premis", "http://www.loc.gov/premis/rdf/v1#");
```

Comment:
More PREMIS v1 for us to worry about later

Reply:
Yes: good eye.